astro/src/content/blog/announcing-fusionauth-1-53.mdx

```diff
@@ -0,0 +1,45 @@
+---
+publish_date: 2024-09-04
+title: Announing FusionAuth Version 1.53.0 - Webhook Logs and New Lambdas
+description: Version 1.53.0 of FusionAuth introduces two hotly-requested features. You can now see a history of webhooks and their statuses, and logins now have more logic.
+authors: Brad McCarty
+image: /img/blogs/release-1-53/fusionauth-1-53-0.png
+categories: Product
+tags: lambdas, login logic, webhook logs, customer data 
+excerpt_separator: "{/* more */}"
+---
+
+We recently released version 1.53.0 of FusionAuth. As we [continue our work](/blog/fusionauth-inc-5000-2024) toward simplification and a better user experience, we've added some hotly-requested features. Marching forward with our animal-based naming convention, I hereby dub version 1.53.0 The Login Lambda Llama.
+
+<center>
```

Comment:
```suggestion
```

Do you need this? Is there a better way to center it? I don't see this tag used anywhere else. Maybe you can make it 1200px wide like all the other images?